pkg/virt-api/webhooks/validating-webhook/validating-webhook_test.go

```diff
@@ -1487,6 +1487,77 @@ var _ = Describe("Validating Webhook", func() {
 			causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vm.Spec)
 			Expect(causes).To(BeEmpty())
 		})
+		It("should allow single multus network with a multus default", func() {
+			vm := v1.NewMinimalVMI("testvm")
+			vm.Spec.Domain.Devices.Interfaces = []v1.Interface{
+				*v1.DefaultNetworkInterface(),
+			}
+			vm.Spec.Domain.Devices.Interfaces[0].Name = "multus1"
+			// 3rd interfaces uses the default pod network, name is "default"
```

Comment:
I am not sure what this comment here and below tries to communicate

Reply:
Another team member wrote these tests. I'll dig around and see if I can figure it out or bug them.

Reply:
I actually might have written these. This was a copy/paste from the test above it that these were based on.

Fixed